lib/build.js

```diff
@@ -42,6 +42,11 @@ function build(clusterWorkerId) {
     }
     str = str + encode(alphabet.lookup, seconds);
 
+    // rebuild if str doesn't end in alpha-numeric character
+    while ( !/[a-zA-Z0-9]$/.test(str) ) {
```

Comment:
This feels like it could be a recurring issue since it impedes the main purpose of the module. 

What would you say to something along these lines?:

```
    // rebuild if str doesn't end in alpha-numeric character
    if ( !/[a-zA-Z0-9]$/.test(str) ) {
        var chars = Array.from( alphabet.characters().replace(/[^a-zA-Z0-9]/, "") );
        var char = chars[Math.floor(Math.random()*chars.length)];
        str = str.slice(0, -1) + char;
    }
```
This would just replace the last character with a random letter from `characters()`